wildfly/commit-markable-resource/src/test/java/org/jboss/narayana/quickstarts/cmr/CmrLrcoTestCase.java

```diff
@@ -50,6 +50,8 @@ public void setup(ManagementClient managementClient, String containerId) throws
 
             try(FailuresAllowedBlock allowedBlock = creaper.allowFailures()) {
                 creaper.execute("/subsystem=transactions/commit-markable-resource=\"java:jboss/datasources/jdbc-cmr\":add()");
+                creaper.execute(
+                        "/subsystem=messaging-activemq/server=default/jms-queue=\"cmr\":add(entries=[java:/queue/cmr])");
```

Comment:
Is the new line needed? 

You can ignore this comment... :see_no_evil:

Reply:
Agreed: we have line length limitations which necessitates adding them, although I forget what it is - when I started out it used to be 80 characters to accommodate the old VT 100 type computer terminals but I think the recommended length is a bit longer now - do we have anything in our checkstylesheet?

Reply:
I set an automatic new line when the line is longer than 120 chars (IIRC) to be more readable.

Reply:
@mmusgrov I cannot see anything related to length limitations here (https://github.com/wildfly/wildfly-checkstyle-config/blob/master/src/main/resources/wildfly-checkstyle/checkstyle.xml). Maybe somewhere else?